marimo/_output/hypertext.py

```diff
@@ -107,6 +113,11 @@ def __init__(self, text: str) -> None:
             self, _hypertext_cleanup, self._virtual_filenames
         )
         finalizer.atexit = False
+        mimetype, data = self._mime_()
+        self.serialized_mime_bundle = {
+            "mimetype": mimetype,
+            "data": data,
+        }
```

Comment:
i think we should move this above the `ctx = get_context()` stuff, right below `self._text = text`